docs/source/bond_family_guide/sysadmin_guide/eula.rst

```diff
@@ -0,0 +1,73 @@
+
+.. _EULA_sysadmin_guide:
+
+***********************************************
+INTEL EVALUATION LICENSE AGREEMENT FOR END USER
```

Comment:
@dcmiddle Can you help us with this issue? Does the old R3 EULA still apply to Bond?